### ISSUE TITLE
BUG : screen width and height swapped in event context

### ISSUE THREAD

**Issue by imjamesim** [rudderlabs/rudder-sdk-ios#301]
**Describe the bug**

1. Screen width and height seem to be swapped in the event context.
2. Sample payload:
```
{
    "anonymousId": "cec60ae6-c020-4c33-8c77-496bf1294ae4",
    "channel": "mobile",
    "context": {
        "app": {
            "build": "1",
            "name": "Flavrs Beta",
            "namespace": "com.flavrs.flavrsapp-beta",
            "version": "1.10.0"
        },
        "device": {
            "attTrackingStatus": 0,
            "id": "cec60ae6-c020-4c33-8c77-496bf1294ae4",
            "manufacturer": "Apple",
            "model": "iPhone",
            "name": "iPhone 14",
            "type": "iOS"
        },
        "library": {
            "name": "rudder-ios-library",
            "version": "1.12.1"
        },
        "locale": "ko-US",
        "network": {
            "carrier": "unavailable",
            "cellular": false,
            "wifi": true
        },
        "os": {
            "name": "iOS",
            "version": "16.1"
        },
        "screen": {
            "density": 3,
            "height": 390,
            "width": 844
        },
        "sessionId": 1681163144,
        "timezone": "America/Los_Angeles",
        "traits": {
            "anonymousId": "cec60ae6-c020-4c33-8c77-496bf1294ae4",
            "userId": "EKRvh462YZxyNvIgkd4IN"
        }
    },
    "event": "Feed",
    "integrations": {
        "All": true
    },
    "messageId": "a2573c7e-2e3a-49a3-8fd8-40b30657a7ee",
    "originalTimestamp": "2023-04-10T21:46:06.367Z",
    "properties": {
        "name": "Feed"
    },
    "rudderId": "6a2bffd2-127f-4c88-98f4-23616b01abe7",
    "sentAt": "2023-04-10T21:46:07.582Z",
    "type": "screen",
    "userId": "EKRvh462YZxyNvIgkd4IN"
}
```

3. The issue repros on all of our events on Rudder iOS SDK v1.12.1.
4. Our SDK is initialized on the main thread on app launch.
5. The issue is not specific to any device model or OS version.
6. There isn't any relevant error stack trace.
7. We first noticed the issue on 4/7/23.

**To Reproduce**
Steps to reproduce the behaviour:

1. Initialise iOS SDK 1.12.1
2. Make sure the device is in portrait mode, and then trigger any log events.
3. See that the width and height are swapped.

**Expected behavior**
The width and height should match the device width and height in the current orientation.

**Screenshots**
N/A

**Version of the _iOS_ SDK**
1.12.1

**Mobile device mode integration**
N/A

**SDK initialisation snippet**
Please provide the following information:

The following SDK initialization happens on app launch on the main thread:

```
let configBuilder = RSConfigBuilder()
    .withDataPlaneUrl(analyticsConfig.rudderDataPlaneURL)
    .withTrackLifecycleEvens(true)
    .withAutoSessionTracking(true)
    .withRecordScreenViews(false)
RSClient.getInstance(analyticsConfig.rudderWriteKey, config: configBuilder.build())
```

**Check for Correct Usage of _writeKey_ and _dataPlaneUrl_**
Confirmed.

**Comment by imjamesim** [comment]
FYI this may be where the width and height may be swapped: https://github.com/rudderlabs/rudder-sdk-ios/blob/b80345a57953cc5d210cf412cf8c2b2a36caa6ec/Sources/Classes/RSScreenInfo.m#L28

**Comment by 1abhishekpandey** [comment]
@imjamesim We fixed this issue in the iOS SDK version `1.13.1`. Please update to this version to resolve the issue.
